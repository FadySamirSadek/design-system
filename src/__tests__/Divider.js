import React from 'react'
import renderer from 'react-test-renderer'
import { Divider, theme } from '..'

describe('Divider', () => {
  test('renders', () => {
    const json = renderer.create(<Divider />).toJSON()
    expect(json).toMatchSnapshot()
  })

  test('borderColor prop sets borderColor', () => {
    const json = renderer.create(<Divider borderColor="accent" />).toJSON()
    expect(json).toMatchSnapshot()
    expect(json).toHaveStyleRule('border-color', 'accent')
  })
})
