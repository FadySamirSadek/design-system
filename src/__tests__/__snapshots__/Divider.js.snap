// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider borderColor prop sets borderColor 1`] = `
.c0 {
  margin-left: 0px;
  margin-right: 0px;
  border: 0;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-color: accent;
}

<hr
  className="c0"
/>
`;

exports[`Divider renders 1`] = `
.c0 {
  margin-left: 0px;
  margin-right: 0px;
  border: 0;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-color: smoke;
}

<hr
  className="c0"
/>
`;
